core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseResourceAccessor.java

```diff
@@ -37,53 +38,75 @@ public class LiquibaseResourceAccessor implements ResourceAccessor {
 	private Resource resource;
 
 	public LiquibaseResourceAccessor(Resource resource) {
-		super();
 		this.resource = resource;
```

Comment:
je hebt geen resource meer nodig, maar de configuration classloader (of scope provider oid) die alleen in zijn eigen scope kijkt of de liquibase file bestaat.